tracer/test/test-applications/integrations/Samples.Wcf/Server/ICalculator.cs

```diff
@@ -4,9 +4,18 @@
 
 namespace Samples.Wcf.Server
 {
+    /// <summary>
+    ///     Note: after updating this interface you'll need to re-generate the proxy class <see cref="CalculatorClient"/>.
+    ///     That can be done in a developer VS prompt (when the WCF server is running):
+    ///         <code>svcutil.exe /language:cs /out:generatedProxy.cs /config:app.config http://localhost:8585/WcfSample/</code>
+    ///     This makes a new file <c>generatedProxy.cs</c> and the contents of this can be swapped out with <see cref="CalculatorClient"/>.
+    /// </summary>
```

Comment:
Nice, thanks for adding this 👍

Reply:
Thank you for adding this, if we ever need to update this again this will be extremely helpful 🙏🏼